src/material/datepicker/date-range-input.ts

```diff
@@ -47,6 +47,7 @@ let nextUniqueId = 0;
     'class': 'mat-date-range-input',
     '[class.mat-date-range-input-hide-placeholders]': '_shouldHidePlaceholders()',
     '[attr.id]': 'null',
+    'role': 'group',
```

Comment:
I was thinking about this for the date-range input too (when I rebased #18528). Did you consider handling the form-field aria label in the group here? I think that would make more sense than doing it per input.

https://www.w3.org/TR/WCAG20-TECHS/ARIA17.html

Reply:
I'm not sure whether setting it on the range input would make much of a difference, because focus won't land on it.

Reply:
It looks like whether the group is tabbale/focusable is not relevant for `role="group"`. The outlined example in W3C actually does not have a focusable group either.

Without a label on the group, screen readers seem to just announce `Edit text group` (ChromeVox), while ideally it would also mention the control's label. Another downside of the approach of using `aria-labelledby` on individual inputs is that screenreaders announce the label whenever you are jumping between the inputs, while the expected experience seems to be that it just announces the (potentially long) group label the first time a group control becomes active.

Given that this seems to be the recommended pattern (and matches the date-range input markup), I think we should consider doing it when we actually care about adding `role="group"`.

Reply:
I just tried it out with NVDA and you're right that this works much better 👍. I've updated it to have the `aria-labelledby` on the range input.